src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.ts

```diff
@@ -145,9 +145,9 @@ export class RgwBucketFormComponent extends CdForm implements OnInit, AfterViewC
       'mfa-token-serial': [''],
       'mfa-token-pin': [''],
       lock_enabled: [{ value: false, disabled: this.editing }],
-      encryption_enabled: [null],
+      encryption_enabled: [{ value: null, disabled: !this.kmsConfigured && !this.s3Configured }], 
```

Comment:
```suggestion
      encryption_enabled: [{ value: null, disabled: !this.kmsConfigured || !this.s3Configured }], 
```
We would like to give select option when any of them is configured not both  cc @nizamial09 @aaSharma14

Reply:
@afreen23 , the condition here intends to disable the check box when both kms and s3 are not configured. Even if one of them is configured the checkbox is enabled. `!this.kmsConfigured && !this.s3Configured` is the correct condition here. `!this.kmsConfigured || !this.s3Configured` will disable the checkbox if either of them is not configured which is incorrcet.

Reply:
@afreen23 may you clarify the approach? I would make changes then